src/library.js

```diff
@@ -277,6 +279,10 @@ LibraryManager.library = {
   vfork: 'fork',
   posix_spawn: 'fork',
   posix_spawnp: 'fork',
+  posix_spawn_file_actions_adddup2: 'fork',
+  posix_spawn_file_actions_addopen: 'fork',
+  posix_spawn_file_actions_destroy: 'fork',
+  posix_spawn_file_actions_init: 'fork',
```

Comment:
Can we move all this stuff to native code at some point?

Reply:
Yes, it could be in C I guess, and then it would work in standalone mode too. Til now though it's been quicker and easier to do this way.

Reply:
Can we merge this to unblock the symbol server work for now, and put a TODO in place to clean this up later?

Reply:
Sure..    we can merge this now.

However I would note that a better solution for porting existing would be to modify that code not to expect these symbols..  any code that actually relies on these symbols is by definition not to fully work on emscripten and could probably benefit from a HAVE_POSIX_SPAWN macro.